### PR TITLE
FEAT: Link text effect

### PR DESCRIPTION
**Pull request by jakubpetrik**
Summary:
- Add new case for TextEffect to support links.
- Bump version

### CODE REVIEW

File: Styles/Classes/TextEffect.swift [hunk 1]
```diff
@@ -80,6 +102,8 @@ extension TextEffect: Equatable {
             return lStyle == rStyle && lMatch.isEqual(rMatch)
         case (.image(let lImage, let lStyle, let lMatch), .image(let rImage, let rStyle, let rMatch)):
             return lImage == rImage && lStyle == rStyle && lMatch.isEqual(rMatch)
+        case (.link(let lURL, let lStyle, let lMatch), .link(let rURL, let rStyle, let rMatch)):
```
Comment (matejdecky) [inline Styles/Classes/TextEffect.swift]: Can this be done via swift 4.2 default implementation of Equatable?

Comment (jakubpetrik) [reply Styles/Classes/TextEffect.swift]: Yes, but we wouldn't be able to use Match as we are using since it will have Self type requirement.

Comment (matejdecky) [reply Styles/Classes/TextEffect.swift]: There would have to be requirement of Equatable protocol for Match protocol and implementation would have to be the identity (===). Well it was just an idea :-)

Comment (jakubpetrik) [reply Styles/Classes/TextEffect.swift]: Yes, but at the same time Match would need to be only as generic constraint which I'm not quite sure we want. Anyway... I'm going to merge this one. Feel free to create a ticket if you think we should convert it. @matejdecky

**Pull request merged**